C/casacorewrapper/build_tarballs.jl

```diff
@@ -0,0 +1,42 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "casacorewrapper"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/kiranshila/casascorewrapper.git", "f7fe573901bb8d651c60fd3f3c23527232de6d57")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd casascorewrapper/
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release ..
+make -j${nproc}
+make install
+exit
+"""
+
+# Use the same platforms from casacore
+platforms = supported_platforms(exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
+platforms = expand_cxxstring_abis(platforms)
+platforms = expand_gfortran_versions(platforms)
```

Comment:
Why?  I don't see any need for this one
```suggestion
```

Reply:
I just copied out of the casacore build. Is there anyway to say "use the platforms this JLL has"?

Reply:
Not really and there is also no need for: you don't need to build this library for different libgfortran versions if there is no fortran involved. Whatever upstream libraries do doesn't matter

Reply:
> Yeah @giordano see, it's still trying to grab libgfortran3 things.

That's because you haven't done what I asked 🙂

Reply:
Ah - I was getting mixed up

Reply:
I get it! Thanks!